src/index.html

```diff
@@ -10,10 +10,280 @@
           minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Document</title>
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.min.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
+  <header class="header">
+      <div class="header-logo">
+        <a href="#"><img src="images/logo.png" alt="mojo"></a>
+      </div>
+      <nav class="nav">
+        <ul class="menu">
+          <li class="menu-item">
+            <a class="menu-link upper-registr is-active" href="#">apple</a>
+          </li>
+          <li class="menu-item">
+            <a class="menu-link upper-registr" href="#">samsung</a>
+          </li>
+          <li class="menu-item upper-registr">
+            <a class="menu-link upper-registr" href="#">Смартфоны и телефоны</a>
+          </li>
+          <li class="menu-item">
+            <a class="menu-link upper-registr" href="#" data-qa="nav-hover">Ноутбуки и компьютеры</a>
+          </li>
+          <li class="menu-item">
+            <a class="menu-link upper-registr" href="#">гаджеты</a>
+          </li>
+          <li class="menu-item">
+            <a class="menu-link upper-registr" href="#">планшеты</a>
+          </li>
+          <li class="menu-item">
+            <a class="menu-link upper-registr" href="#">фото</a>
+          </li>
+          <li class="menu-item">
+            <a class="menu-link upper-registr" href="#">видео</a>
+          </li>
+      </ul>
+    </nav>
+  </header>
+  <main class="main">
+    <div class="catalog-card">
+      <div class="product-card" data-qa="card">
+        <div class="product-img">
+          <img class="product-icon" src="./images/imac.jpeg" alt="product_card">
```

Comment:
icon is considered to be something small, not the image describing the product. Change the class name